src/ralph/data_importer/mixins.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.contenttypes.models import ContentType
+from ralph.data_importer.models import ImportedObjects
+
+
+class ImportForeignKeyMixin(object):
+
+    """ImportForeignKeyMixin class for django import-export resources."""
+
+    def before_save_instance(self, instance, dry_run):
+        self.old_object_pk = instance.pk
+        instance.pk = None
```

Comment:
what about importing with fixed id/pk? Didn't we discuss it that pk shouldn't be cleaned and old ralph should export id in other column?